### ISSUE TITLE
Source for Browser?

### ISSUE THREAD

**Issue by Godzil** [autc04/executor#24]
Do we have the sources for the default Browser?

If we don't have the source of the current (and so being able to rebuild it as needed) it would probably be worth to make a new one which is Open Source.

**Comment by MaddTheSane** [comment]
https://github.com/autc04/executor/tree/master/packages/go Is the source code of the Browser.

**Comment by Godzil** [comment]
Oh I missed it because the resources don't match. That's interesting, that also mean the sources we have don't match the pre-buile one.

For exemple the sources says:

```
iGo, by Bill Goldman
Copyright ARDI 1994, 1995
send bug reports to:
browser-bugs@ardi.com
Last modified ^0
```

Which is not the about box it shows.
Hard to tell if it's just resource editing between both or some unknown changes which are not in the repository

**Comment by autc04** [comment]
Yes, I am afraid that the browser source code that was published is not entirely up to date.

So far, I've failed twice (read: lost patience and abandoned the project) at making it compile.
It's old (pre-ANSI) C code, with  an average of three pointer casts per line. It freely mixes model and view, and still ends up needing more memory than Apple's Finder. I seriously doubt that anyone will ever want to extend and improve it.

I like the idea of just implementing a new one - it would be a nice application of my other project (github.com/autc04/Retro68) and might also function as an excuse and testbed for implementing Drag Manager support in Executor.
On the other hand, it might be argued that what we really need is good integration with various native file managers, but I think that's quite hard, as well.
